transports/sqs/native-integration_native-message-attributes-promotion_sqs_[7.3,).partial.md

```diff
@@ -0,0 +1,6 @@
+### Native message attributes promotion
+
+When ingesting native messages, e.g., messages sent by a non-NServiceBus sender, any native message SQS attribute will be promoted by the SQS transport to an NServiceBus header.
```

Comment:
```suggestion
When ingesting native messages, e.g., messages sent by a non-NServiceBus sender, the SQS transport will promote all native message SQS attributes to an NServiceBus header.
```

Do we also want to clarify where those will be? Either in the payload or in an SQS native header containing all nsb headers? Or is that out of scope here?

Reply:
I think it's not in scope here